src/modules/mavlink/mavlink_parameters.cpp

```diff
@@ -111,6 +111,17 @@ MavlinkParametersManager::handle_message(const mavlink_message_t *msg)
 				strncpy(name, set.param_id, MAVLINK_MSG_PARAM_VALUE_FIELD_PARAM_ID_LEN);
 				/* enforce null termination */
 				name[MAVLINK_MSG_PARAM_VALUE_FIELD_PARAM_ID_LEN] = '\0';
+
+				/* Whatever the value is, we're being told to stop sending */
+				if (strncmp(name, "_HASH_CHECK", sizeof(name)) == 0) {
+					char buf[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN];
+					sprintf(buf, "[pm] hash load stopped param listing");
```

Comment:
That's completely fair, this snuck in as a missed debugging update precisely _because_ I'm used to ignoring vehicle message spam.